addon/templates/components/validated-input.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if (not-eq type "checkbox")}}
-  {{#if label}}
-      <label class={{config.css.label}} for={{inputId}}>
-        {{label}}{{#if required}} {{requiredLabel}}{{/if}}
-      </label>
+  {{#if (or label (not-eq labelComponent 'validated-label'))}}
+    <span style="display: inline-block;">
```

Comment:
Ember is wrapping component render in  `<div id="ember..." class="ember-view"> </div>`, but we want our label components to be inline with our input. So I have to add an `inline-block` style :wink:

Reply:
Why not just use `tagName: 'label'` on the `validated-label` component and remove it from the template? The `for` and `class` can be easily mapped too. This results in a cleaner DOM since we have way less elements..

Reply:
Cool, I will do that!